Interni/Verbali Interni/2020_12_18/VI_2020_12_14.tex

```diff
@@ -0,0 +1,22 @@
+\documentclass[a4paper, oneside]{article}
```

Comment:
Guarda su informazioni.tex cosa ho scritto, TLDR cambia il nome di questo file in VI_2020_12_19